### PR TITLE
Add CSSTransition API data

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds data for the CSSTransition API based upon manual testing.  This API is a part of a [W3C draft](https://drafts.csswg.org/css-transitions-2/#the-CSSTransition-interface).

### CODE REVIEW

File: api/CSSTransition.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+  "api": {
+    "CSSTransition": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:7]:
```suggestion
            "version_added": "78"
```

Comment (jpmedley) [reply api/CSSTransition.json:7]: https://storage.googleapis.com/chromium-find-releases-static/d69.html#d692c061c5a4f07611af04d53a032c8ac4dde7c5

File: api/CSSTransition.json [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          },
+          "chrome_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:10]:
```suggestion
            "version_added": "78"
```

File: api/CSSTransition.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+          },
+          "edge": {
+            "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:13]:
```suggestion
            "version_added": "≤79"
```

File: api/CSSTransition.json [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            "version_added": "13.1"
+          },
+          "safari_ios": {
+            "version_added": "13.4"
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:40]:
```suggestion
            "version_added": "78"
```

File: api/CSSTransition.json [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "CSSTransition": {
+        "__compat": {
+          "description": "<code>CSSTransition()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:54]:
```suggestion
              "version_added": "78"
```

File: api/CSSTransition.json [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:57]:
```suggestion
              "version_added": "78"
```

File: api/CSSTransition.json [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+            },
+            "edge": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:60]:
```suggestion
              "version_added": "≤79"
```

File: api/CSSTransition.json [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:87]:
```suggestion
              "version_added": "78"
```

File: api/CSSTransition.json [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "transitionProperty": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:101]:
```suggestion
              "version_added": "78"
```

File: api/CSSTransition.json [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:104]:
```suggestion
              "version_added": "78"
```

File: api/CSSTransition.json [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+            },
+            "edge": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:107]:
```suggestion
              "version_added": "≤79"
```

File: api/CSSTransition.json [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+              "version_added": "13.1"
+            },
+            "safari_ios": {
+              "version_added": "13.4"
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/CSSTransition.json:134]:
```suggestion
              "version_added": "78"
```

**Comment by vinyldarkscratch** [comment]
I see why my tests were off on this one.  Strangely enough, it seems Firefox has a constructor implemented, even though that's not defined by the spec (and also why I thought Chrome had no support).  I removed the constructor data from the file, but considering this, would it be better to leave it in and mark it as non-standard, @Elchi3?

**Comment by ddbeck** [comment]
The [spec says](https://drafts.csswg.org/css-transitions-2/#the-CSSTransition-interface) `CSSTransition` inherits from `Animation` which _does_ have a constructor. If the browsers are implementing it, we ought to have data for it and I think it's standards track.

File: api/CSSTransition.json (hunk 5 above)
Comment (ddbeck) [reply api/CSSTransition.json:54]: @vinyldarkscratch There are three resolved, but unapplied suggestions here. Was this a mistake (maybe relating to the revert) or was there more information that conflicted with version 78 for these entries?

**Comment by jpmedley** [comment]
You should ping the spec authors to be sure. It's possible the spec is out of date with some standards resolution. It doesn't happen often, but it does happen.

File: api/CSSTransition.json (hunk 5 above)
Comment (vinyldarkscratch) [reply api/CSSTransition.json]: The lack of merge was a mistake -- I forgot to commit the suggestions when adding the constructor code back.  Fixed now!

**Pull request merged**